backend/api.test/Database/DatabaseUtilities.cs

```diff
@@ -124,8 +124,8 @@ public async Task<Robot> NewRobot(RobotStatus status, Area area, Installation in
                 IsarId = Guid.NewGuid().ToString(),
                 RobotType = RobotType.Robot,
                 SerialNumber = "0001",
-                CurrentInstallation = installation,
-                CurrentArea = area,
+                CurrentInstallationCode = installation.InstallationCode,
+                AreaName = area.Name,
```

Comment:
Could maybe have it be CurrentAreaName to keep consistent

Reply:
In CreateRobotQuery.cs I mean

Reply:
Changed it in `CreateRobotQuery.cs`